spec/truffle.mspec

```diff
@@ -148,8 +153,7 @@ class MSpecScript
   set :files, get(:language) + get(:core) + get(:library) + get(:truffle)
 end
 
-is_child_process = ENV.key? "MSPEC_RUNNER"
-if i = ARGV.index('slow') and ARGV[i-1] == '--excl-tag' and is_child_process
+if i = ARGV.index('slow') and ARGV[i-1] == '--excl-tag' and MSpecScript.child_process?
```

Comment:
I don't know if we have a Rubocop rule for this, but I thought we settled on `&&` for simple boolean expressions like this.